modules/doc/content/getting_started/installation/conda.md

```diff
@@ -11,9 +11,9 @@ and an experimental [WSL](installation/windows10.md) option is available.
 
 !include installation/install_conda_moose.md
 
-!include getting_started/installation/clone_moose.md
+!include getting_started/installation/clone_moose.md optional=True
```

Comment:
So this change is for Griffin specifically. We want to include the conda install documentation, but the cloning and testing moose documentation is not necessary since we are cloning griffin and updating the submodule. It's not a big deal though, I don't mind reverting this change.

Reply:
That's fine to leave as is; I just needed to understand the rationale.